lighthouse-core/gather/driver.js

```diff
@@ -46,11 +46,17 @@ class Driver {
     this._devtoolsLog = new DevtoolsLog(/^(Page|Network)\./);
     connection.on('notification', event => {
       this._devtoolsLog.record(event);
-      this.recordNetworkEvent(event.method, event.params);
+      this._recordNetworkEvent(event.method, event.params);
       this._eventEmitter.emit(event.method, event.params);
     });
     this.online = true;
     this._domainEnabledCounts = new Map();
+
+    /**
+     * Used for monitoring network status events.
+     * @private {?NetworkRecorder}
+     */
+    this._networkStatus = null;
```

Comment:
I'm not in love with this name change, network status makes me thing it's only watching for offline/online or something yet we listen for individual requests, dispatch all network messages to it, etc. Thoughts?